src/lib/platform/MSWindowsDesks.cpp

```diff
@@ -602,6 +606,16 @@ MSWindowsDesks::deskThread(void* vdesk)
 {
     MSG msg;
 
+	BOOL vistaOrGreater = FALSE;
+
+	{
```

Comment:
strange indentation

Reply:
Yeah, the indentation is a mess; the same file uses both spaces and tabs and its hard to tell which is which. I'll try to make it more consistent.

Reply:
I think it makes sense to gradually move to spaces. Various tools display tabs differently, using spaces works around this issue very well.

Reply:
True. Luckily it doesn't matter anymore in this case, since this was just replaced by a Windows function. :D